src/MassTransit/Logging/Diagnostics/LogContextActivityExtensions.cs

```diff
@@ -266,7 +266,24 @@ static ActivityContext GetParentActivityContext(Headers headers, bool isRemote =
             activity.SetTag(DiagnosticHeaders.Messaging.Operation, "process");
 
             if (activity.IsAllDataRequested)
+            {
+                if (context.MessageId.HasValue)
+                    activity.SetTag(DiagnosticHeaders.MessageId, context.MessageId.Value.ToString("D"));
+                if (context.ConversationId.HasValue)
+                    activity.SetTag(DiagnosticHeaders.Messaging.ConversationId, context.ConversationId.Value.ToString("D"));
+                if (context.CorrelationId.HasValue)
+                    activity.SetTag(DiagnosticHeaders.CorrelationId, context.CorrelationId.Value.ToString("D"));
+                if (context.RequestId.HasValue)
+                    activity.SetTag(DiagnosticHeaders.RequestId, context.RequestId.Value.ToString("D"));
+                if (context.InitiatorId.HasValue)
+                    activity.SetTag(DiagnosticHeaders.InitiatorId, context.InitiatorId.Value.ToString("D"));
+                if (context.SourceAddress != null)
+                    activity.SetTag(DiagnosticHeaders.SourceAddress, context.SourceAddress.ToString());
```

Comment:
otel has a required `source.address` attribute, which it different than this one, but essential to identify the broker  (it's usually broker's FQDN).  I suggest to set it.

Reply:
Should we use `context.SourceAddress.Host` (`SourceAddress is a `Uri`) in this case to only set the FQDN? Should we also update the send activity?

Reply:
It's worthless in RabbitMQ, however, since it doesn't have the virtual host name, or the port.